app/models/market_model.rb

```diff
@@ -0,0 +1,17 @@
+class MarketModel < ActiveRecord::Base
+  include Privacy
+
+  FOUNDATIONS = ["kWh", "kW_connection", "kW_max", "kW_contracted", "kW_flex"]
```

Comment:
I think we're going to start out with just one of these, and then add the others as the sprint progresses. I think we should be starting with `kW_max` or `kWh` but I haven't decided yet (it'll be whichever is easiest :laughing:). It's okay for us to keep the full list so long as the client understands that only one of them will work for the time being; otherwise we should probably narrow it down to "kW_max".